packages/utils/src/internals/general.ts

```diff
@@ -5,21 +5,15 @@ import { setTimeout } from 'node:timers/promises';
  * Default regular expression to match URLs in a string that may be plain text, JSON, CSV or other. It supports common URL characters
  * and does not support URLs containing commas or spaces. The URLs also may contain Unicode letters (not symbols).
  */
-// eslint-disable-next-line
-export const URL_NO_COMMAS_REGEX = RegExp(
-    'https?://(www\\.)?[\\p{L}0-9][-\\p{L}0-9@:%._\\+~#=]{0,254}[\\p{L}0-9]\\.[a-z]{2,63}(:\\d{1,5})?(/[-\\p{L}0-9@:%_\\+.~#?&//=\\(\\)]*)?',
-    'giu',
-);
+export const URL_NO_COMMAS_REGEX =
```

Comment:
Not even syntastic could help me uncover what the actual difference is here because of the switch to `RegExp` literals. Could you highlight it for me somehow, please? :smile:

Reply:
the previous version required "at least one char followed by anything followed by at least one char" in the host name, the new one also allows "only one char"

```diff
-(www\\.)?[\\p{L}0-9][-\\p{L}0-9@:%._\\+~#=]{0,254}[\\p{L}0-9]\\.[a-z]{2,63}
+(www\\.)?([\\p{L}0-9]|[\\p{L}0-9][-\\p{L}0-9@:%._\\+~#=]{0,254}[\\p{L}0-9])\\.[a-z]{2,63}
```

here is simplified version with just `\w` so its more visible

```diff
-(www\.)?\w[\w-0-9@:%._\+~#=]{0,254}\w\.[a-z]{2,63}
+(www\.)?(\w|\w[\w-0-9@:%._\+~#=]{0,254}\w)\.[a-z]{2,63}
```